pkg/jx/cmd/test_data/step_create_task/from_yaml/pipeline.yml

```diff
@@ -5,30 +5,40 @@ metadata:
   name: abayer-js-test-repo-really-long-1
   namespace: jx
 spec:
-  params: []
+  params:
+    - default: 0.0.1
+      description: the version number for this pipeline which is used as a tag on docker
+        images and helm charts
+      name: version
   resources:
-  - name: abayer-js-test-repo-really-long
-    type: git
+    - name: abayer-js-test-repo-really-long
+      type: git
   tasks:
-  - name: build-a-really-long-stage-name-please-but-not-too-long-thanks
-    resources:
-      inputs:
-      - name: workspace
-        resource: abayer-js-test-repo-really-long
-      outputs:
-      - name: workspace
-        resource: abayer-js-test-repo-really-long
-    taskRef:
-      name: abayer-js-test-repo-really-long-build-a-really-long-stage-nam-1
-  - name: second
-    resources:
-      inputs:
-      - from:
+    - name: build-a-really-long-stage-name-please-but-not-too-long-thanks
+      params:
+        - name: version
+          value: ${params.version}
+      resources:
+        inputs:
+          - name: workspace
+            resource: abayer-js-test-repo-really-long
+        outputs:
+          - name: workspace
+            resource: abayer-js-test-repo-really-long
+      taskRef:
+        name: abayer-js-test-repo-really-long-build-a-really-long-stage-nam-1
+    - name: second
+      params:
+        - name: version
+          value: ${params.version}
+      resources:
+        inputs:
+          - from:
+              - build-a-really-long-stage-name-please-but-not-too-long-thanks
+            name: workspace
+            resource: abayer-js-test-repo-really-long
+      runAfter:
         - build-a-really-long-stage-name-please-but-not-too-long-thanks
-        name: workspace
-        resource: abayer-js-test-repo-really-long
-    runAfter:
-      - build-a-really-long-stage-name-please-but-not-too-long-thanks
-    taskRef:
-      name: abayer-js-test-repo-really-long-second-1
-status: {}
+      taskRef:
+        name: abayer-js-test-repo-really-long-second-1
+status: {}
```

Comment:
For what's its worth, there is a newline missing. Same for some more files in this commit.